packages/ui/v2/core/Dialog.tsx

```diff
@@ -175,12 +141,32 @@ export function DialogHeader(props: DialogHeaderProps) {
 export function DialogFooter(props: { children: ReactNode }) {
   return (
     <div>
-      <div className="mt-5 flex justify-end space-x-2 rtl:space-x-reverse">{props.children}</div>
+      <div className="mt-7 flex justify-end space-x-2 rtl:space-x-reverse ">{props.children}</div>
     </div>
   );
 }
 
 DialogContent.displayName = "DialogContent";
 
 export const DialogTrigger = DialogPrimitive.Trigger;
-export const DialogClose = DialogPrimitive.Close;
+// export const DialogClose = DialogPrimitive.Close;
+
+export function DialogClose(
+  props: {
+    dialogCloseProps?: React.ComponentProps<typeof DialogPrimitive["Close"]>;
+    children?: ReactNode;
+    onClick?: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+    disabled?: boolean;
+    color?: ButtonProps["color"];
+  } & React.ComponentProps<typeof Button>
+) {
+  const { t } = useLocale();
+  return (
+    <DialogPrimitive.Close asChild {...props.dialogCloseProps}>
+      {/* This will require the i18n string passed in */}
+      <Button color={props.color || "minimal"} {...props}>
+        {props.children ? props.children : t("Close")}
+      </Button>
+    </DialogPrimitive.Close>
+  );
+}
```

Comment:
Create a wrapper for dialogclose - no point doing 
```
<DialogClose>
	<Button />
</DialogClose> 
```
Every single time we need this